jobs/build-arch.Jenkinsfile

```diff
@@ -376,14 +357,16 @@ lock(resource: "build-${params.STREAM}-${basearch}") {
             }
         }
 
+        // A few independent tasks that can be run in parallel
+        def parallelruns = [:]
+
```

Comment:
can probably even bury this inside the if statement below since the usage is only in that scope now.